core/ui/src/androidTest/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardTest.kt

```diff
@@ -79,7 +79,7 @@ class NewsResourceCardTest {
         }
 
         composeTestRule
-            .onNodeWithText(dateFormatted)
+            .onNodeWithText(dateFormatted, substring = true)
```

Comment:
This test previously was asserting that only the date was displayed for unknown news resource types. Now there's no such thing as an unknown type so we should remove this test entirely.

Reply:
Well... it depends. Previously, the check was on the `Unknown` enum. But I replaced it with the `"Unknown"` string. The same check (that would make the test pass without any change) is based on an 'empty' String comparison.
Any empty String for type currently exists in prod.
What should we do?

- updating the test data with an empty (unknown) type String
- removing the test altogether
- keeping everything as-is?

Reply:
Good point. The Card behaviour has changed so the test should change to exercise that code path. Let's go with the first option, update the test data to have an empty type string. 

Let's also update the test name to `testMetaDataDisplay_withEmptyResourceType`.